### DISCUSSION TITLE
마무리 수정사항 논의

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/check-in-listener#44] [General]
앱이 거의 마무리되었기 때문에 (에러 해결 문제가 남았으니 많은 참여 부탁드립니다 #39 )
마무리 수정사항을 논의하고자 합니다.
@euphony-io/eu-us  팀원분들 각자 다운받으셔서 테스트해보시고,

아이콘 크기 변경 제안이나 저희 앱 안의 '~ 버튼을 눌러주세요' 등의 텍스트 변경 의견도 좋습니다.
파일 추출 완성 시점에서 토스트메시지 추가해보는 것도 좋을 것 같고요!

수정사항이 없는 것 같으면 없다는 코멘트도 좋습니다. 모두 리뷰 부탁드려요 :)

**Comment by sujin9** [comment]
좋아요! 파일 추출할 때 사용자에게 파일 경로 안내 해주면서 파일 생성 완료 토스트 메시지 발생시키는 것도 좋을 것 같아요
제가 파일 추출 경로 수정해서 같이 PR 올릴게요 !
다른 부분은 저는 다 괜찮은 것 같아요 :)

**Reply by Jsueeee** [reply]
예쓰 좋아요~👍👍

**Reply by dlwlstks96** [reply]
좋은 아이디어입니다!!👍

**Comment by dlwlstks96** [comment]
내일 리드미 작성과 함께 어플 최종 테스트하며 개선 방안이 있을지 검토해보겠습니다:)

**Comment by sujin9** [comment]
저희 앱 지금 보면 마이크 / 저장 공간 권한 없이 버튼 누르면 안내 문구 뜨는거 영어로 되어 있잖아요!
혹시 이거 한국어로 바꿔야할까요???
지금 토스트 메시지 수정 중인데 필요하면 그것도 같이 수정할게요!

**Reply by YoungSeokHong** [reply]
넵 그게 좋을 것 같습니다!

**Reply by Jsueeee** [reply]
넵 잘하셨어요!! 👍👍👍
